tournaments/index.js

```diff
@@ -100,23 +101,23 @@ class Tournament {
 		this.generator.getUsers().forEach(user => {
 			let error = generator.addUser(user);
 			if (typeof error === 'string') {
-				output.sendReply('|tournament|error|' + error);
+				output.sendReply(`|tournament|error|${error}`);
 				isErrored = true;
 			}
 		});
 
 		if (isErrored) return;
 
 		this.generator = generator;
-		this.room.send('|tournament|update|' + JSON.stringify({generator: generator.name}));
+		this.room.send(`|tournament|update|{"generator": "${generator.name}"}`);
```

Comment:
This is going to crash if `generator.name` has a `"` in it.

Just keep using `JSON.stringify`.